test/components/embedders/test_azure_document_embedder.py

```diff
@@ -257,4 +275,7 @@ def test_run(self):
             assert isinstance(doc.embedding, list)
             assert len(doc.embedding) == 1536
             assert all(isinstance(x, float) for x in doc.embedding)
-        assert metadata == {"model": "text-embedding-ada-002", "usage": {"prompt_tokens": 15, "total_tokens": 15}}
+
+        assert metadata["usage"]["prompt_tokens"] == 15
+        assert metadata["usage"]["total_tokens"] == 15
+        assert "ada" in metadata["model"]
```

Comment:
unrelated: tests were failing locally (with main too) due to "ada" in the response instead of "text-embedding-ada-002".
(I found a similar test relaxation [here](https://github.com/deepset-ai/haystack/blob/e1e797206d05912a2e41ee40d05a42518686f11f/test/components/embedders/test_openai_document_embedder.py#L254))